easybuild/easyconfigs/o/OpenEXR/OpenEXR-2.3.0-intel-2018b.eb

```diff
@@ -15,6 +15,8 @@ checksums = ['8243b7de12b52239fe9235a6aeb4e35ead2247833e4fbc41541774b222717933']
 
 builddependencies = [('CMake', '3.12.1')]
 
+dependencies = [('Boost', '1.67.0')]
```

Comment:
I saw the error that caused this was
```
boost Python is a required dependency when OPENEXR_BUILD_PYTHON_LIBS is set
```
so perhaps we should actually be unsetting `OPENEXR_BUILD_PYTHON_LIBS` or you should be including Python as a dependency.

Reply:
@smoors Why not leave `OPENEXR_BUILD_PYTHON_LIBS` enabled and include `Boost.Python` as a dep? Seems like minimal effort...

That would also require the customary `versionsuffix` of course (and including `Python` as a direct dep to ensure `%(pyver)s` template is resolved).

Reply:
I could try that, but Boost.Python is version 1.68.0 for 2018b (annoyingly), whereas OpenImageIO and Blender use Boost 1.67.0 (Blender recommends this version).
it works as it is now, and I prefer not to add Python as it is not needed.